packages/lab/stories/dropdown-next/dropdown-next.doc.stories.mdx

```diff
@@ -9,22 +9,29 @@ import {
 
 # Dropdown
 
-A Dropdown allows the user to select an item from an array of options displayed in an overlay list– with the selected value displayed in the dropdown field.
+A Dropdown allows the user to select an item from an array of options displayed in an overlay list–with the selected value displayed in the dropdown field.
 Use the Dropdown component when you have a limited list of options (5-10) that don’t require filtering to narrow down.
 
 Dropdown contains a Button with a chevron Icon, that opens a List on mouse and keyboard interactions.
-When `defaultSelected` prop is passed, Dropdown displays its value on initial render, otherwise it’s empty by default.
 
 <Canvas>
   <Story id="lab-dropdown-next--default" />
 </Canvas>
 
-### Secondary
+### With `defaultSelected` prop
+
+When `defaultSelected` prop is passed, Dropdown displays its value on initial render, otherwise it’s empty by default.
```

Comment:
Let's make it clear it's meant for uncontrolled. Refer to react [`input` doc](https://react.dev/reference/react-dom/components/input#props). Adjust props's JSDoc if needed

Reply:
updated docs and stories